ui/src/getPackageInfo.ts

```diff
@@ -5,60 +5,54 @@
 
 import * as fse from 'fs-extra';
 import { ExtensionContext } from "vscode";
-import { IActionContext } from "../index";
-import { callWithTelemetryAndErrorHandling } from "./callWithTelemetryAndErrorHandling";
 import { ext } from "./extensionVariables";
-import { parseError } from "./parseError";
 
-export function getPackageInfo(ctx?: ExtensionContext): { extensionName: string, extensionVersion: string, aiKey: string, extensionId: string, bugsUrl: string | undefined } {
-    if (!ctx) {
-        ctx = ext.context;
-    }
+interface IPackageInfo {
+    extensionName: string;
+    extensionVersion: string;
+    aiKey: string;
+    extensionId: string;
+    bugsUrl: string | undefined;
+}
 
-    let packageJson: IPackageJson = {};
-    // tslint:disable-next-line:no-floating-promises
-    callWithTelemetryAndErrorHandling('azureTools.getPackageInfo', (context: IActionContext) => {
```

Comment:
This was causing timing problems, because of this call stack:
> registerUIExtensionVariables -> createTelemetryReporter -> getPackageInfo -> callWithTelemetryAndErrorHandling

Basically, it tries to get the extension variables, while registering/creating the extension variables. In the past we've been able to work around this, but I decided it just wasn't worth the complexity after I hit more problems. I took a look at telemetry and very few people had errors, most of which were "Too many files open". I feel like that error is much more helpful to users than the error they're seeing now, which is just "Extension's package.json is missing aiKey"